src/status_im/native_module/core.cljs

```diff
@@ -133,9 +137,9 @@
   (.verify (status) address hashed-password callback))
 
 (defn login-with-keycard
-  [{:keys [whisper-private-key encryption-public-key on-result]}]
+  [{:keys [multiaccount-data password chat-key]}]
```

Comment:
could you add a docstring that explains what this does as well as a description of the arguments, particularly chat-key